src/components/device/ha-device-trigger-picker.ts

```diff
@@ -0,0 +1,148 @@
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-item/paper-item-body";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu-light";
+import "@polymer/paper-listbox/paper-listbox";
+import memoizeOne from "memoize-one";
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  css,
+  CSSResult,
+  property,
+} from "lit-element";
+import { HomeAssistant } from "../../types";
+import { fireEvent } from "../../common/dom/fire_event";
+import computeStateName from "../../common/entity/compute_state_name";
+import {
+  DeviceTrigger,
+  fetchDeviceTriggers,
+  triggersEqual,
+} from "../../data/device_automation";
+
+class HaDeviceTriggerPicker extends LitElement {
+  public hass?: HomeAssistant;
+  @property() public label?: string;
+  @property() public deviceId?: string;
+  @property() public triggers: DeviceTrigger[] = [];
+  @property() public trigger?: DeviceTrigger;
+  @property() public presetTrigger?: DeviceTrigger;
+  private noTrigger: DeviceTrigger = {
```

Comment:
If it's always the same, put it above the class definition.

Reply:
it's not always the same, it's updated here:
https://github.com/home-assistant/home-assistant-polymer/blob/35a50333a56aa521542cbb63c5cdcbeb1a02faa5/src/components/device/ha-device-trigger-picker.ts#L88-L89
But maybe I misunderstand your point?

Reply:
good point, leave this as is.